tools/vsextension/ProjectWizard/WizardImplementation.cs

```diff
@@ -0,0 +1,296 @@
+                    string token = trimmed.Split(' ', ':', '?')[0];
+
+                    if (token == "CFG_TA_FLOAT_SUPPORT")
+                    {
+                        // "CFG_TA_FLOAT_SUPPORT := y" means hardware float support is enabled.
+                        hardwareFloatSupported = trimmed.EndsWith("y");
+                    }
+                    else if (token == "sm")
+                    {
+                        // "sm := ta_arm32" means 32-bit.
+                        is32Bit = trimmed.EndsWith("ta_arm32");
+                    }
+                }
+            }
+            catch (IOException)
+            {
+                // Output a warning so the developer knows they won't get hardware float support.
+                IVsOutputWindow outWindow = Package.GetGlobalService(typeof(SVsOutputWindow)) as IVsOutputWindow;
+                Guid generalPaneGuid = VSConstants.GUID_OutWindowGeneralPane;
+                IVsOutputWindowPane generalPane;
+                outWindow.GetPane(ref generalPaneGuid, out generalPane);
+                if (generalPane == null)
+                {
+                    // The General pane isn't there yet, so create it.
+                    string customTitle = "General";
+                    outWindow.CreatePane(ref generalPaneGuid, customTitle, 1, 1);
+                    outWindow.GetPane(ref generalPaneGuid, out generalPane);
+                }
+                if (generalPane != null)
+                {
+                    generalPane.OutputString("Warning: No hardware float support detected, using software support instead");
+                    generalPane.Activate(); // Bring the pane into view.
+                }
+            }
+        }
+
+        // Convert a path in Windows format to a path in WSL format.
+        private string GetUnixPath(string folder)
+        {
+            string root = Path.GetPathRoot(folder).ToLower();
+            string relativeFolder = folder.Substring(root.Length).Replace('\\', '/');
+            string drive = root.Substring(0, 1);
+            string unixPath = "/mnt/" + drive + "/" + relativeFolder;
+            return unixPath;
+        }
+
+        /// <summary>
+        /// Set the path to the OpenEnclave libs and TA Dev Kit, as required for ARM builds.
+        /// </summary>
+        /// <param name="replacementsDictionary">macro dictionary to update</param>
+        /// <param name="isWindows">true if creating a Windows project</param>
+        /// <returns>false if canceled, true otherwise</returns>
+        private bool SetOELibPath(Dictionary<string, string> replacementsDictionary, bool isWindows)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            if (isWindows)
+            {
+                // No ARM support implemented yet for Windows.
+                return true;
+            }
+
+            // First try picking the board from a menu.
+            BoardPickerPage picker = new BoardPickerPage();
```

Comment:
Should this be shown still?

Reply:
Per discussion, this only shows up for a Linux project; support for that is coming later.